packages/vue/src/tracing.ts

```diff
@@ -75,7 +75,7 @@ export const createTracingMixins = (options: TracingOptions): Mixins => {
               this.$_sentryRootSpan ||
               activeTransaction.startChild({
                 description: 'Application Render',
-                op: VUE_OP,
+                op: 'ui.vue.render',
```

Comment:
Let's change this for [consistency](https://github.com/getsentry/sentry-javascript/blob/7304215d875decf0bf555cab82aa90fc1341b27e/packages/vue/src/tracing.ts#L103) to:

```suggestion
                op: `${VUE_OP}.render`,
```